content/en/about/day-in-life-dev.md

```diff
@@ -6,95 +6,111 @@ position: 7
+Design reviews are basically code reviews at a higher level. New projects and large features usually start with a design
+doc and review. Design docs are a great tool for gathering early feedback and encouraging discussion between developers
+and business stakeholders before anything gets set in stone. A doc is a lot easier to change than code, and code is
+easier to change than a running system. Conveniently, it also serves as great documentation for new hires or anyone else
+wondering “why was X done this way?” A typical doc will answer questions like:
 
-- Why are we building this? How does it benefit the business / team? Are we making any assumptions about our customers and future product direction?
+- Why are we building this? How does it benefit the business / team? Are we making any assumptions about our customers
+  and future product direction?
 - Is there any existing work we are taking advantage of? Why or why not?
-- What are the tradeoffs we are making? e.g., do we care more about space consumption or speed? Does this feature need a new RDS instance, or will a tiny DynamoDB table work  just as well?
-- What is the requirement for uptime? How can we tell if something is wrong? How screwed are we if S3 goes down, and are we OK with that?
-- The design doc is written by whoever owns the feature in question. Typically, initial review happens via comment threads in Google Docs, and is followed by a quick Hangouts session to make sure all outstanding issues are resolved before proceeding.
+- What are the tradeoffs we are making? e.g., do we care more about space consumption or speed? Does this feature need a
+  new RDS instance, or will a tiny DynamoDB table work just as well?
+- What is the requirement for uptime? How can we tell if something is wrong? How screwed are we if S3 goes down, and are
+  we OK with that?
+- The design doc is written by whoever owns the feature in question. Typically, initial review happens via comment
+  threads in Notion, and is followed by a quick Hangouts session to make sure all outstanding issues are resolved
+  before proceeding.
 
-Writing the design document sometimes involves prototyping and experimenting with technologies, depending on the nature of the work.
+Writing the design document sometimes involves prototyping and experimenting with technologies, depending on the nature
+of the work.
 
-## A typical calendar
+## A Typical Calendar
 
-We attempt to avoid [swiss cheesing our calendars](https://thinkingthrough.substack.com/p/stop-swiss-cheesing-your-calendar) 
+We attempt to
```

Comment:
Left this section as is while we re-jigger our meetings. Hopefully not too untrue.